bravado/async_http_client.py

```diff
@@ -44,111 +33,16 @@ def start_request(self, request_params):
 
```

Comment:
Github won't let me place this comment where I want (line 34 pre-commit, line 23 post-commit) but I think we need to stop beating around the bush and creating additional "naming" layers where they're not necessary.  So:

s/AsynchronousHttpClient/FidoHttpClient/g
s/SynchronousHttpClient/RequestsHttpClient/g

Ok to defer to #120